### PR TITLE
Non-Backward-Compatible Change to YAML Renderer

### PR DESCRIPTION
**Pull request by kartnico**
For this issue : #3

### CODE REVIEW

File: nut/server.sls [hunk 1]
```diff
@@ -20,7 +20,7 @@ ups_conf:
     - name: {{ nut.config_dir }}/{{ nut.server.ups.config }}
     - source: salt://nut/templates/ups-conf.jinja
     - context:
-        ups: {{ ups_config }}
+        ups: {{ ups_config|tojson }}
```
Comment (myii) [inline nut/server.sls]:
```suggestion
        ups: {{ ups_config | json }}
```

Further info: https://github.com/saltstack-formulas/consul-formula/pull/40#issuecomment-472728517.

File: nut/server.sls [hunk 2]
```diff
@@ -37,7 +37,7 @@ upsd_conf:
     - name: {{ nut.config_dir }}/{{ nut.server.upsd.config }}
     - source: salt://nut/templates/upsd-conf.jinja
     - context:
-        upsd: {{ upsd_config }}
+        upsd: {{ upsd_config|tojson }}
```
Comment (myii) [inline nut/server.sls]: Likewise:

```suggestion
        upsd: {{ upsd_config | json }}
```

File: nut/server.sls [hunk 3]
```diff
@@ -54,7 +54,7 @@ upsd_users:
     - name: {{ nut.config_dir }}/{{ nut.server.users.config}}
     - source: salt://nut/templates/upsd-users.jinja
     - context:
-        users: {{ upsd_users }}
+        users: {{ upsd_users|tojson }}
```
Comment (myii) [inline nut/server.sls]: Likewise:

```suggestion
        users: {{ upsd_users | json }}
```

**Comment by javierbertoli** [comment]
@kartnico any change you can submit the changes @myii suggest? As he mentions, we found (while dealing with this same issue in other formulas) that `| json` has the same effect that `| tojson` and is backward compatible.

**Comment by myii** [comment]
Merged, thanks @kartnico and also @javierbertoli.

**Pull request merged**